src/modules/Elsa.MongoDb/Serializers/PolymorphicSerializer.cs

```diff
@@ -0,0 +1,57 @@
+using MongoDB.Bson;
+using MongoDB.Bson.IO;
+using MongoDB.Bson.Serialization;
+using MongoDB.Bson.Serialization.Serializers;
+
+namespace Elsa.MongoDb.Serializers;
+
+public class PolymorphicSerializer : IBsonSerializer<object>
+{
+    public Type ValueType => typeof(object);
+
+    public object Deserialize(BsonDeserializationContext context, BsonDeserializationArgs args)
+    {
+        var reader = context.Reader;
+        var bookmark = reader.GetBookmark();
```

Comment:
Initially, I got confused about the term "bookmark" (we use it in the Elsa domain). But I see that this is a clever way provided by the MongoDB package to bookmark the current position in the reader. Neat!